plugins/ImageGraph/API.php

```diff
@@ -238,10 +238,12 @@ public function get(
 
             $ordinateColumns = array();
             if (empty($columns)) {
-                if (!empty($reportColumns[self::DEFAULT_ORDINATE_METRIC])) {
+                if (!empty($metadata['metrics']) && false) {
+                    foreach ($metadata['metrics'] as $key => $value) {
+                        $ordinateColumns[] = $key;
+                    }
+                } else if (!empty($reportColumns[self::DEFAULT_ORDINATE_METRIC])) {
                     $ordinateColumns[] = self::DEFAULT_ORDINATE_METRIC;
-                } else if (!empty($metadata['metrics'])) {
-                    $ordinateColumns[] = key($metadata['metrics']);
```

Comment:
@AltamashShaikh I'm not sure if this is the optima solution. This will actually always show all available metrics. For a report with many metrics this might be unwanted.
The method actually has a parameter `$columns`, which can be used to pass the columns that should be printed.
So if at some place a metric might be missing, it might maybe be better to adjust it to pass the list of columns to display.

Reply:
also I haven't looked at this but `&& false` in an if statement is suspicious

Reply:
@justinvelluppillai was testing something before push and I pushed this by mistakenly